jsx/Filter.js

```diff
@@ -53,6 +53,9 @@ class Filter extends Component {
         case 'select':
           element = <SelectElement key={filter.name} options={filter.options}/>;
           break;
+        case 'multiselect':
+          element = <SelectElement key={filter.name} options={filter.options} multiple={true}/>;
+          break;
```

Comment:
Does this work? I'm not sure the filter is currently set up to handle mutliple inputs and there may need to be changes made to `onFieldUpdate` for this to work.

Reply:
@HenriRabalais  Seems to be working; anyway I will test it again. The value is a result of a GROUP_CONCAT query. returning `GROUP_CONCAT(DISTINCT modality.ID) as sequenceType`